test/src/test/java/org/corfudb/infrastructure/LayoutServerTest.java

```diff
@@ -464,6 +522,53 @@ public void testReboot() throws Exception {
         }
     }
 
+    @Test
+    public void testChangeEpochInPhase2ByBaseServer() {
+        String serviceDir = PARAMETERS.TEST_TEMP_DIR;
+        ServerContext sc = new ServerContextBuilder()
+                .setPort(0)
+                .setMemory(false)
+                .setLogPath(serviceDir)
+                .setServerRouter(getRouter())
+                .build();
+
+        // set up base server
+        BaseServer bs = new BaseServer(sc);
+
+        IServerRouter routerMock = Mockito.mock(IServerRouter.class);
+        ChannelHandlerContext chcMock = Mockito.mock(ChannelHandlerContext.class);
+
+        // set up layout server
+        LayoutServer ls = new LayoutServer(sc);
```

Comment:
Pavel, just one note -- having more descriptive variable names would be great. Using two letter variable names makes the code quite unreadable IMO.

Reply:
Fixed

Reply:
Thanks!